sdk/batch/microsoft-azure-batch/CHANGELOG.md

```diff
@@ -1,14 +1,34 @@
 # Release History
 
-## 11.2.0-beta.1 (Unreleased)
+## 11.2.0 (2024-09-10)
 
 ### Features Added
 
-### Breaking Changes
+- Force delete/terminate job or job schedule:
```

Comment:
Are these breaking changes acceptable for track 1 libraries?

Reply:
Got confirmation that our breaking changes got approved at the Swagger level so there shouldn't be any issue at the SDK level